windows/devices/uwb/UwbDeviceConnector.cxx

```diff
@@ -103,14 +103,14 @@ UwbConnector::GetDeviceInformation()
     // the case, the first attempt will succeed. Otherwise, the buffer is grown
     // to account for the vendor specific information, and the IOCTL attempted a
     // second time.
-    for (const auto i : std::ranges::iota_view{ 1, 2 }) {
+    for (const auto i : std::ranges::iota_view{ 0, 2 }) {
         deviceInformationBuffer.resize(bytesRequired);
         PLOG_DEBUG << "IOCTL_UWB_GET_DEVICE_INFO attempt #" << i << " with " << std::size(deviceInformationBuffer) << "-byte buffer";
         BOOL ioResult = DeviceIoControl(handleDriver.get(), IOCTL_UWB_GET_DEVICE_INFO, nullptr, 0, std::data(deviceInformationBuffer), std::size(deviceInformationBuffer), &bytesRequired, nullptr);
         if (!LOG_IF_WIN32_BOOL_FALSE(ioResult)) {
             DWORD lastError = GetLastError();
             // Treat all errors other than insufficient buffer size as fatal.
-            if (lastError != ERROR_INSUFFICIENT_BUFFER) {
+            if (lastError != ERROR_MORE_DATA) {

```

Comment:
Could we perhaps leave these changes for another PR? This would require immediate changes to the CX, which I'd like to avoid until we can have a discussion with Davut as well

Reply:
Yep, I started a discussion offline. We'll want to close on this quickly, though, since we rely on the simulator for testsing, and this is going to become increasingly important once we move up the stack toward the true service portions.

Reply:
Tagging @davutkarabay.

Reply:
This one is synchronous, so I've left it. However, I reverted the one for `IOCTL_UWB_NOTIFICATION` which is overlapped/async, so will resolve this comment.